src/jit/liveness.cpp

```diff
@@ -2184,6 +2184,7 @@ void Compiler::fgComputeLifeLIR(VARSET_TP& life, BasicBlock* block, VARSET_VALAR
 #if defined(FEATURE_SIMD)
             case GT_SIMD_CHK:
 #endif // FEATURE_SIMD
+            case GT_JCMP:
```

Comment:
This affects life.  I looked at the `default` case, but it looked to me like this was the correct choice.

Reply:
Looks right to me